generic/openblas.py

```diff
@@ -49,14 +50,24 @@ def setUp(self):
         openblas_dir = os.path.join(self.workdir, "OpenBLAS-develop")
         openblas_bin_dir = os.path.join(openblas_dir, 'bin')
         os.mkdir(openblas_bin_dir)
-        build.make(openblas_dir, extra_args='FC=gfortran '
-                   'ISMAXKERNEL=imax.S ISMINKERNEL=imin.S')
+        cmd = "gcc --version"
+        output = process.system_output(cmd, shell=True).decode("utf-8")
+        gcc_ver = re.search(r'\d+', output)
```

Comment:
although this works can we narrow it down to `output.splitlines()[0]` so that the execution time reduces?